src/components/InputBook.js

```diff
@@ -36,29 +36,28 @@ const InputBook = () => {
     }
   };
   return (
-    <div>
-      <form>
-        <label htmlFor="test">
-          ADD NEW BOOK
-          <input type="text" name="title" placeholder="Book title" value={title || ''} onChange={(e) => setTitle(e.target.value)} />
-          <input type="text" name="author" placeholder="Author" value={author || ''} onChange={(e) => setAuthor(e.target.value)} />
-          <select type="text" name="category" value={category} onChange={(e) => setCategory(e.target.value)} required>
-            <option value="" disabled selected>
-              select Category
-            </option>
-            <option>Fiction</option>
-            <option>Horror</option>
-            <option>Romance</option>
-            <option>Science</option>
-            <option>NonFiction</option>
-          </select>
-        </label>
+    <>
+      <hr className="input-line" />
+      <h3 className="input-header"> ADD NEW BOOK</h3>
```

Comment:
- [ ]  Kindly it will be better 😃 if you put the correct color for the text here to make it nice like the original design as you see here it is different from the original design, so let's make it like the zeplin design 💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://github.com/Jeddaa/bookstore/assets/83360636/361916b3-e880-412b-b9a5-af9fd9c1fd2d)   |  ![image](https://github.com/Jeddaa/bookstore/assets/83360636/0a4361a4-790c-4be5-bef4-2c327478b55f)  |